src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  let array = [];
```

Comment:
Avoid reassigning the variable values

Reply:
Avoid using the data structure name in the variable name. It's kind of strange :)